create-neon/src/bin/create-neon.ts

```diff
@@ -53,7 +61,14 @@ async function main(name: string) {
       versions: inferVersions()
     });
   }
-
+  try{
+    // setting PWD back to working directory shouldn't be required
+    await fs.rename(tmpDirPath +'/'+name, workingDirPath+'/'+name)
```

Comment:
Please use `path.join` instead of concatenating strings. It would handle using the proper escaping and separators on all operating systems.